apps/testdsa.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 1998-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1998-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Since testdsa.h is changed due to function refactore in PR #3656 , so I decide to leave this (should bump up year in that PR, but forgot...)